staging/src/k8s.io/client-go/features/known_features.go

```diff
@@ -37,6 +37,10 @@ const (
 	//  The feature is disabled in Beta by default because
 	//  it will only be turned on for selected control plane component(s).
 	WatchListClient Feature = "WatchListClient"
+
+	// owner: @nilekhc
+	// alpha: v1.30
+	StorageVersionMigrator Feature = "StorageVersionMigrator"
```

Comment:
didn't notice before, this collides with the pkg/features name.  You need a different name here: `InformerResourceVersion`?